modules/ROOT/pages/endpoint-https-send.adoc

```diff
@@ -42,18 +42,27 @@ Enter a well-formed URL (including the path) that specifies where to send the re
 `+https://www.mulesoft.com/sendRequest?name=something+`
 |x
 
+|Response Timeout (in milliseconds)
+|Maximum time, in milliseconds, for which the request element blocks the execution of the flow that is waiting for the HTTP response. The defauilt is 15000 milleseconds.
+|x
+
+|Retry interval (in seconds)
+|Frequency of reconnection attempts when unable to connect to the remote HTTP or HTTPS service. The default is 60 seconds.
+|x
+
+|Maximum retry attempts
+|Maximum number of reconnection attempts to make. If you don’t want Partner Manager to retry after an initial connection failure, set this value to `0`. The default is `3`.
```

Comment:
```suggestion
|Maximum number of reconnection attempts. To prevent Partner Manager from retrying after an initial connection failure, set this value to `0`. The default is `3`.
```